src/middleware/http-errors.ts

```diff
@@ -0,0 +1,13 @@
+import { Context, Middleware, Next } from 'koa';
```

Comment:
`middleware/http-errors` has the same base name as the library `http-errors`, possible confusion when looking at the imported module in the middle of other files. Is it the most fitting name?